bucket/mednafen.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.24.1",
+    "homepage": "https://mednafen.github.io/",
+    "description": "Emulator for different systems",
+    "license": "GNU GPLv2",
+    "architecture": {
+        "64bit": {
+            "url": "https://mednafen.github.io/releases/files/mednafen-1.24.1-win64.zip",
+            "hash": "ad3a7906521f0ce45737c2f0f7ea1117799548e60d10a746ac74ddd26d944cbb"
+        },
+        "32bit": {
+            "url": "https://mednafen.github.io/releases/files/mednafen-1.24.1-win32.zip",
+            "hash": "d26456eb711feac679e424968edee579b69c38ce1972a93bed10355b04ff21cd"
+        }
+    },
+    "bin": "mednafen.exe",
+    "shortcuts": [
+        [
+            "mednafen.exe",
+            "Mednafen"
+        ]
+    ],
+    "persist": [
+        "cheats",
+        "firmware",
+        "mcm",
+        "mcs",
+        "palettes",
+        "pgconfig",
+        "sav",
+        "snaps"
+        ]
```

Comment:
```suggestion
        "snaps",
        "mednafen.cfg"
    ],
    "checkver": {
        "url": "https://mednafen.github.io/documentation/ChangeLog.txt",
        "regex": "--\\s+([\\d.]+):\\s+--"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://mednafen.github.io/releases/files/mednafen-1.24.1-win64.zip"
            },
            "32bit": {
                "url": "https://mednafen.github.io/releases/files/mednafen-1.24.1-win32.zip"
            }
        },
        "hash": {
            "url": "https://mednafen.github.io/",
            "regex": "(?sm)$basename.*?SHA-256:\\s*$sha256"
        }
    }
```

Reply:
On another note, is there a decided style to be used for the files or is it arbitrary? Do we have something like a clang-format or something to use?

Reply:
What style?

Everything needed is specified editorconfig and validation is done in CI

Reply:
https://ci.appveyor.com/project/lukesampson/scoop-extras/builds/32344679

![A](https://i.imgur.com/k5cpRSa.png)

Reply:
Oh, okay. Thank you.